_pages/production.md

```diff
@@ -72,8 +72,8 @@ Once you have:
 1. [Confirmed that this integration is listed in a signed IAA](#confirm-iaa)
 2. [Created a production configuration](#production-configuration)
 
-Please submit the [login.gov production integration request form](https://share.hsforms.com/1UTxHGOu2Q0SVyz9ulknZGw3ak9e){:target='_blank'}.
+Please submit the [login.gov production integration request form](https://app.smartsheetgov.com/b/form/68d06187790f4e18a5dd13e932703f7c){:target='_blank'}.
```

Comment:
is it worth defining a reference for this link so we don't have to find/replace? (and update the other location to use `[smartsheet]` too?

```suggestion
Please submit the [login.gov production integration request form][smartsheet]{:target='_blank'}.

[smartsheet]: https://app.smartsheetgov.com/b/form/68d06187790f4e18a5dd13e932703f7c
```

Reply:
They're two different forms but I like this a lot - I'll update and merge. Thanks!